file_writer.go

```diff
@@ -211,7 +211,7 @@ func (f *FileWriter) Close() error {
 	}
 
 	// Retry on Complete request returning false
-	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
```

Comment:
this is not what I meant.
I meant that the `backOff` value itself should be  capped so we don't wait too long between retries.